### ISSUE TITLE
service-worker.js not found 404

### ISSUE THREAD

**Issue by FDiskas** [mozilla-neutrino/react-starter#11]
neutrino v6.1.1
here is a gist https://gist.github.com/FDiskas/1271ec490b74faf13a352c1e595d0691

Then I start `npm start` everything works ok - react app is running. In developer tools - network tab I see that service-worker.js file can't be loaded.

and `npm run build` the results are broken app
```
Uncaught Error: Minified React error #37; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=37 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
```
Runned build again and got it working.

**Comment by helfi92** [comment]
Ensure that your index.jsx is inside your `src` directory. By default, Neutrino presets expect all project source code to live in a directory named src in the root of the project.

Also, you don't seem to have service-worker.js in your directory. If you could post a screenshot of your entire dev-tools, then maybe we could look at it more.

**Comment by FDiskas** [comment]
Yes that index.jsx is in src directory. About service-worker.js - I don't have this. Don't know what should be inside that file. :incoming_envelope: maybe some example in the readme file needed.
